components/match_ticker/commons/match_ticker.lua

```diff
@@ -467,6 +479,21 @@ function MatchTicker.switchOpponents(match)
 	return match
 end
 
+--- Fetches region of a tournament
+---@param tournamentPage string
+---@return string?
+MatchTicker.fetchRegionOfTournament = FnUtil.memoize(function(tournamentPage)
+	local tournamentData = mw.ext.LiquipediaDB.lpdb('tournament', {
+		conditions = '[[pagename::' .. tournamentPage .. ']]',
+		query = 'region',
```

Comment:
```suggestion
		query = 'locations',
```
right?

Reply:
yes, was still testing (hence the draft status)